src/parser/preprocessor.c

```diff
@@ -2670,6 +2675,26 @@ void add_define_macro(char const *const name, char const *const macro_arg,
 	def->token_list     = obstack_finish(&pp_obstack);
 }
 
+
+void define_special_macro_functions()
```

Comment:
- How about `define_pragma_macro()`?
- Use `(void)` (`()` unfortunately means "unknown parameter count", rather than "no parameters" in C)